scripts/postinstall.js

```diff
@@ -20,13 +22,17 @@ try {
 
 function setupAutocomplete() {
   return new Promise((resolve, reject) => {
+    let tabtabPath = './node_modules/tabtab';
+    if (!fileExistsSync(path.join(__dirname, '..', 'node_modules', 'tabtab', 'src', 'cli.js'))) {
+      tabtabPath = '../tabtab';
```

Comment:
If you'd use the "get-installed-path" module as I mentioned in #3792, we could skip the tabtab invocation at all.

Only checking the main path and then trying the alternative path without checking it will exist, will trigger the exc-eption later. It would be better to use the module and immediately emit a warning, that tabtab was not found without trying to execute a command that might not exist.

get-installed-path would even find a global tabtab if the local one is not found.